rasa/core/featurizers/tracker_featurizers.py

```diff
@@ -509,7 +509,7 @@ def persist(self, path: Union[Text, Path]) -> None:
         )
 
     @staticmethod
-    def load(path: Text) -> Optional["TrackerFeaturizer"]:
+    def load(path: Union[Text, Path]) -> Optional["TrackerFeaturizer"]:
```

Comment:
I really wish we just stuck to one throughout the code base 😂

Reply:
once we've done the revamp we can prioritize using `Path` everywhere 😆